src/pages/PrepareCondition/index.tsx

```diff
@@ -89,7 +89,7 @@ export const PrepareCondition = () => {
         logger.log(`Condition ID ${conditionId}`)
         if (!conditionExists) {
           const tx = await CTService.prepareCondition(questionId, oracleAddress, numOutcomes)
-          await provider.waitForTransaction(tx)
+          await provider.waitForTransaction(tx, 4)
```

Comment:
It may be a good idea to have this number as a configuration variable and use it in all the `waitForTransaction`.

Reply:
Done it!